population/client.go

```diff
@@ -271,6 +279,59 @@ func (c *Client) GetAreas(ctx context.Context, input GetAreasInput) (GetAreasRes
 	return areas, nil
 }
 
+func (c *Client) GetArea(ctx context.Context, input GetAreaInput) (Area, error) {
+
+	logData := log.Data{
+		"method":          http.MethodGet,
+		"population_type": input.PopulationType,
+		"area_type":       input.AreaType,
+		"area_type_id":    input.AreaTypeID,
+	}
+	urlPath := fmt.Sprintf("population-types/%s/area-types/%s/areas/%s", input.PopulationType, input.AreaType, input.AreaTypeID)
```

Comment:
Why `urlPath` instead of just `url?`

Reply:
its the standard set in the other methods in that file.

`func (c *Client) GetAreas(ctx context.Context, input GetAreasInput) (GetAreasResponse, error) {
	logData := log.Data{
		"method":       http.MethodGet,
		"dataset_id":   input.DatasetID,
		"area_type_id": input.AreaTypeID,
		"text":         input.Text,
	}

	urlPath := fmt.Sprintf("population-types/%s/area-types/%s/areas", input.DatasetID, input.AreaTypeID)`

for example.

Reply:
Fair enough, I'd say that elsewhere it's a slightly bad choice too. I guess it's not a huge thing but it's a Go paradigm to choose shorter variable names wherever possible, you'll find that other organisations will be much more attentive to these kind of details than we tend to be here. It's a good habit to learn the idioms and preferred styles of the language. https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
No im totally okay with what is there already. 
There is a url package that is quite popular that is probably going to clash with your autosuggest if you try to use that as a variable name, same with path btw. 
I'd rather keep `urlPath`

Reply:
will definitely read through the best practice link though for future!